inference/predictor.py

```diff
@@ -28,12 +34,10 @@ def tokenize_inputs(self, text):
                 text,
                 return_tensors="pt",
                 padding="max_length",
-                max_length=max_input_len(input_token_len),
+                max_length=max_input_len(input_token_len),  # noqa: F821 # Undefined name
```

Comment:
@kira-lin `input_token_len` is got by
<img width="309" alt="image" src="https://github.com/intel/llm-on-ray/assets/97155466/0f2a4b76-a958-4969-97a0-96cac4810b54">
and it's removed in [llm-ray#116](https://github.com/intel-sandbox/llm-ray/pull/166). How should I fix it?

Reply:
synced with @kira-lin, remove this first.